app/views/pages/home.html.erb

```diff
@@ -37,7 +37,7 @@
           <p>Find teacher training options if you live in:</p>
           <ul class="govuk-list govuk-list--bullet">
             <li><a href="https://teachinscotland.scot">Scotland</a></li>
-            <li><a href="https://www.discoverteaching.wales/routes-into-teaching/">Wales</a></li>
+            <li><a href="https://gov.wales/becoming-teacher">Wales</a></li>
```

Comment:
Suggest we link to `https://educators.wales/` instead as this is the Wales link used on GIT from 

* https://getintoteaching.education.gov.uk/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk
* https://getintoteaching.education.gov.uk/train-to-teach-in-england-as-an-international-student

Reply:
And is also the site that the discoverteaching.wales link redirects to